lib/active_record/connection_adapters/mysql2rgeo/schema_statements.rb

```diff
@@ -40,8 +40,9 @@ def create_table_definition(*args, **options)
 
         # override
         def new_column_from_field(table_name, field, _definitions)
-          type_metadata = fetch_type_metadata(field[:Type], field[:Extra])
-          default, default_function = field[:Default], nil
+          field["Type"] = 'geometrycollection' if field["Type"] == 'geomcollection'
```

Comment:
The changes in this method to use strings rather than symbols when accessing data in `field` is necessary to work with Rails 8